src/widgets/dialogs/UserInfoPopup.cpp

```diff
@@ -414,51 +415,60 @@ void UserInfoPopup::installEvents()
     std::shared_ptr<bool> ignoreNext = std::make_shared<bool>(false);
 
     // ignore
-    QObject::connect(
-        this->ui_.ignore, &QCheckBox::stateChanged,
-        [this, ignoreNext, hack](int) mutable {
-            if (*ignoreNext)
-            {
-                *ignoreNext = false;
-                return;
-            }
-
-            this->ui_.ignore->setEnabled(false);
-
-            auto currentUser = getApp()->accounts->twitch.getCurrent();
-            if (this->ui_.ignore->isChecked())
-            {
-                currentUser->ignoreByID(
-                    this->userId_, this->userName_,
-                    [=](auto result, const auto &message) mutable {
-                        if (hack.lock())
-                        {
-                            if (result == IgnoreResult_Failed)
-                            {
-                                *ignoreNext = true;
-                                this->ui_.ignore->setChecked(false);
-                            }
-                            this->ui_.ignore->setEnabled(true);
-                        }
-                    });
-            }
-            else
-            {
-                currentUser->unignoreByID(
-                    this->userId_, this->userName_,
-                    [=](auto result, const auto &message) mutable {
-                        if (hack.lock())
-                        {
-                            if (result == UnignoreResult_Failed)
-                            {
-                                *ignoreNext = true;
-                                this->ui_.ignore->setChecked(true);
-                            }
-                            this->ui_.ignore->setEnabled(true);
-                        }
-                    });
-            }
-        });
+    QObject::connect(this->ui_.ignore, &QCheckBox::stateChanged,
```

Comment:
check how the follow connection works and replicate its behaviour

Reply:
or add a todo to do that

Reply:
I am frustrated. For whatever reason this stuff doesn't want to work even though I replicated behavior of follow connection in 100%, since this is kinda out of scope of migrating Kraken methods, I'll do that in a separate PR. Added a TODO :+1: